spec/views/content_items/index.html.erb_spec.rb

```diff
@@ -1,23 +1,27 @@
 require 'rails_helper'
 
 RSpec.describe 'content_items/index.html.erb', type: :view do
-  let(:content_items) { build_list(:content_item, 2) }
+  let(:organisation) { create(:organisation_with_content_items, content_items_count: 4) }
```

Comment:
Yeah i increased it to get a bigger sample size for testing, especially for sorting. This was a personal choice